pkgs/by-name/si/simulide/package.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, stdenv
+, fetchbzr
+, libsForQt5
+}:
+
+let
+  version = "0.4.15";
+  release = "SR10";
+  branch = "simulide_0.4.14"; # the branch name does not mach the version for some reason
+  rev = "291";
+  sha256 = "sha256-BBoZr/S2pif0Jft5wrem8y00dXl08jq3kFiIUtOr3LM=";
+in
+stdenv.mkDerivation {
+  pname = "simulide";
+  version = "${version}-${release}";
+
+  src = fetchbzr {
+    url = "https://code.launchpad.net/~arcachofo/simulide/${branch}";
+    inherit rev sha256;
+  };
+
+  postPatch = ''
+    sed -i resources/simulide.desktop \
+      -e "s|^Exec=.*$|Exec=simulide|" \
+      -e "s|^Icon=.*$|Icon=simulide|"
+    sed -i SimulIDE.pro \
+      -e "s|^VERSION = .*$|VERSION = ${version}|" \
+      -e "s|^RELEASE = .*$|RELEASE = -${release}|" \
+      -e "s|^REV_NO = .*$|REV_NO = ${rev}|" \
+      -e "s|^BUILD_DATE = .*$|BUILD_DATE = ??-??-??|"
+  '';
+
+  preConfigure = ''
+    cd build_XX
+  '';
+
+  nativeBuildInputs = [
+    libsForQt5.qmake
+    libsForQt5.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    libsForQt5.qtserialport
+    libsForQt5.qtmultimedia
+    libsForQt5.qttools
+    libsForQt5.qtscript
+  ];
```

Comment:
```suggestion
  buildInputs = with libsForQt5; [
    qtserialport
    qtmultimedia
    qttools
    qtscript
  ];
```

Reply:
After writing this, I agree now that this looks better